src/cascade/input_data/configuration/local_cache.py

```diff
@@ -0,0 +1,26 @@
+from cascade.core import getLoggers
+
+CODELOG, MATHLOG = getLoggers(__name__)
+
+
+class LocalCache:
+    """A small in-memory cache object. This has the same interface
+    as memcached, so that we can switch it out for that, if appropriate."""
+    def __init__(self, maxsize=1):
```

Comment:
maxsize=1 seems like a very odd default. Why? It doesn't turn the cache off but it also doesn't allow it to be useful.

Reply:
I increased it. It's meant for large things, and it's super dangerous b/c it will increase memory over time. This should be memcached, or go to a file, or explicitly purged. The 1 is a measure of my discomfort. :) It's now 8.